src/libraries/System.Net.Security/tests/StressTests/SslStress/docker-compose.yml

```diff
@@ -1,13 +1,17 @@
 version: '3'
 services:
   client:
-    image: ${SSLSTRESS_IMAGE:-sslstress}
+    build:
+      context: ../../../../ # ~> src/libraries
+      dockerfile: ./System.Net.Security/tests/StressTests/SslStress//${DOCKERFILE:-Dockerfile}
     links:
       - server
     environment:
       - SSLSTRESS_ARGS=--mode client --server-endpoint server:5001 ${SSLSTRESS_CLIENT_ARGS}
   server:
-    image: ${SSLSTRESS_IMAGE:-sslstress}
+    build:
+      context: ../../../../ # ~> src/libraries
```

Comment:
we should share variables for these paths in the root yml variables file.

Reply:
also you could consider something like /src/libraries.

Reply:
That's a docker-compose yml, not azdo yml :-)

Reply:
oh I missed that. Can you still use absolute paths here?

Reply:
Not really, by convention it's relative to the docker-compose.yml